android-client-sdk/src/test/java/com/devcycle/sdk/android/api/EventQueueTests.kt

```diff
@@ -36,7 +36,7 @@ class EventQueueTests {
     @Test
     fun `events are aggregated correctly`() {
         val request = Request("some-key", "http://fake.com", "http://fake.com")
-        val user = User.builder().withUserId("test").build()
+        val user = User.builder().withUserId("test").withIsAnonymous(false).build()
```

Comment:
hm is this actually necessary? I thought we made it so that it would implicitly set isAnonymous to false if you set a user id

Reply:
withUserId isn't setting `isAnonymous` to false in the builder. I can change that if its the better alternative, but as of right now this line errors without the `withIsAnonymous` call

Reply:
ah so its set for the DVCUser class, but this is being instantiated as a base User. It seems like the User class is always created from a DVCUser in the code anyway